services/web/server/src/simcore_service_webserver/garbage_collector/_core_orphans.py

```diff
@@ -157,7 +157,7 @@ async def remove_orphaned_services(
         _logger.debug("Could not fetch list_dynamic_services")
 
     _logger.info(
-        "Currently running services %s",
+        "Listing currently running services (this does not mean I am removing them) %s",
```

Comment:
hahah something like "Observing services" is shorter and as clear as this message :-)